pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/CmdClusters.java

```diff
@@ -130,9 +132,15 @@ void run() throws PulsarAdminException {
                         getAdmin().namespaces().deleteNamespace(namespace, true);
                     }
                     getAdmin().tenants().deleteTenant(tenant);
+                } else {
+                    Set<String> clusters = getAdmin().tenants().getTenantInfo(tenant).getAllowedClusters();
+                    clusters.remove(cluster);
+                    getAdmin().tenants().updateTenant(tenant, TenantInfo.builder()
```

Comment:
If the `clusters` only have one element (`@param cluster`), should we delete this tenant?

/cc @codelipenghui @Technoboy-

Reply:
IMO, we should throw an error since  tenant with empty clusters is not allowed. @Technoboy-  @poorbarcode